docs/usage/supported_k8s_versions.md

```diff
@@ -4,8 +4,9 @@ Currently, the Gardener supports the following Kubernetes versions:
 
 ## Garden cluster version
 
-:warning: The minimum version of the garden cluster that can be used to run Gardener is **`1.10.x`**.
+:warning: The Kubernetes version of a garden cluster that can be used to run Gardener must be in the range of **`1.10.x`** to  **`1.15.x`**.
 The reason for that is that the least supported Kubernetes version in Gardener is `1.10`.
+Furthermore, Gardener's [extension API server](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/) does not support version >= `1.16` yet.
```

Comment:
Out of curiosity: Why not? Because of the admission webhooks? In this case, can you enhance [this code](https://github.com/gardener/gardener/blob/master/pkg/operation/garden/garden.go#L299-L306)?

Reply:
Hm, I thought because of #1605?

Reply:
Ah, right, thanks for the reminder. :)